test/screenshot/top-app-bar/actionItems.ts

```diff
@@ -0,0 +1,12 @@
+export interface actionItem {
+  icon: string;
+  label: string;
+}
+
+const actionItems: actionItem[] = [
```

Comment:
can you change this to just bookmark? Then I think all screenshots will pass. We can add the other 2 later.

Reply:
done